ports/esp32/mpthreadport.c

```diff
@@ -41,7 +41,7 @@
 #define MP_THREAD_DEFAULT_STACK_SIZE                    (MP_THREAD_MIN_STACK_SIZE + MICROPY_STACK_CHECK_MARGIN)
 #define MP_THREAD_PRIORITY                              (ESP_TASK_PRIO_MIN + 1)
 
-#if ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 2, 0) && !CONFIG_FREERTOS_ENABLE_STATIC_TASK_CLEAN_UP
+#if !CONFIG_FREERTOS_ENABLE_STATIC_TASK_CLEAN_UP
```

Comment:
Is this if logic needed at all now that this option is always disabled?

Reply:
Oh yeah, have pulled all this out!